plugins/net_plugin/net_plugin.cpp

```diff
@@ -2726,21 +2724,19 @@ namespace eosio {
       std::lock_guard<std::mutex> g( chain_info_mtx );
       chain_lib_num = cc.last_irreversible_block_num();
       chain_lib_id = cc.last_irreversible_block_id();
-      chain_head_blk_num = cc.head_block_num();
-      chain_head_blk_id = cc.head_block_id();
-      chain_fork_head_blk_num = cc.fork_db_pending_head_block_num();
-      chain_fork_head_blk_id = cc.fork_db_pending_head_block_id();
-      fc_dlog( logger, "updating chain info lib ${lib}, head ${head}, fork ${fork}",
-               ("lib", chain_lib_num)("head", chain_head_blk_num)("fork", chain_fork_head_blk_num) );
+      chain_fork_head_blk_num = cc.fork_db_head_block_num();
+      chain_fork_head_blk_id = cc.fork_db_head_block_id();
+      fc_dlog( logger, "updating chain info lib ${lib}, fork ${fork}",
+               ("lib", chain_lib_num)("fork", chain_fork_head_blk_num) );
    }
 
-   //         lib_num, head_blk_num, fork_head_blk_num, lib_id, head_blk_id, fork_head_blk_id
-   std::tuple<uint32_t, uint32_t, uint32_t, block_id_type, block_id_type, block_id_type>
+   //        lib_num, fork_head_blk_num, lib_id, fork_head_blk_id
+   std::tuple<uint32_t, uint32_t, block_id_type, block_id_type>
    net_plugin_impl::get_chain_info() const {
       std::lock_guard<std::mutex> g( chain_info_mtx );
       return std::make_tuple(
-            chain_lib_num, chain_head_blk_num, chain_fork_head_blk_num,
-            chain_lib_id, chain_head_blk_id, chain_fork_head_blk_id );
+            chain_lib_num, chain_fork_head_blk_num,
+            chain_lib_id, chain_fork_head_blk_id );
```

Comment:
This would work as well: `return { chain_lib_num, chain_fork_head_blk_num, chain_lib_id, chain_fork_head_blk_id };`.

Reply:
Done.